sycl/plugins/level_zero/pi_level_zero.hpp

```diff
@@ -371,7 +372,7 @@ struct _pi_device : _pi_object {
   bool ImmCommandListsPreferred;
 
   // Return whether to use immediate commandlists for this device.
-  bool useImmediateCommandLists();
+  int useImmediateCommandLists();
```

Comment:
move https://github.com/intel/llvm/pull/7041/files#diff-15dd1eb076d2164bd9e87d9057f05f652a716498e8cdf5975e564c65309a0985R765-R773 enum here and return it

Reply:
Ok, done.